lib/response_massager.js

```diff
@@ -6,9 +6,9 @@ class ResponseMassager {
     this.elasticSearchResponse = responseReceived
   }
 
-  massagedResponse (request) {
+  massagedResponse (request, itemUri) {
```

Comment:
We're passing in `itemUri` solely to tell it to live-query the customer code. What do you think about tightening that contract so it's called via `massagedResponse (request, { queryCustomerCode: true })`, obligating `_fixItemAvailabilityInResponse` to be defined like

```
 (barcodesAndAvailability, request, options = {}) {
   options = Object.assign({
     queryCustomerCode: false // Optionally live-query SCSB for latest customer code
   }, options)
```

It's functionally the same in this case, but 1) clarifies the purpose of param, and 2) gives us a place to add other optional params in the future